Anytype/Sources/PresentationLayer/Modules/Membership/TierSelection/MembershipTierSelectionView.swift

```diff
@@ -45,8 +47,10 @@ struct MembershipTierSelectionView: View {
                     }
                 case .appStore(let product):
                     MembershipNameSheetView(tier: model.tierToDisplay, anyName: model.userMembership.anyName, product: product, onSuccessfulPurchase: model.onSuccessfulPurchase)
-                case .external:
-                    EmptyView() // TBD in future updates
+                case .external(let info):
+                    StandardButton("More info", style: .primaryLarge) {
```

Comment:
Localization?

Reply:
It is temporary design for testing purposes